src/rpc/misc.cpp

```diff
@@ -151,24 +153,28 @@ UniValue validateaddress(const UniValue& params, bool fHelp)
 {
     if (fHelp || params.size() != 1)
         throw runtime_error(
-            "validateaddress \"zcashaddress\"\n"
-            "\nReturn information about the given Zcash address.\n"
+            "validateaddress \"zenaddress\"\n"
+            "\nVerify an ZEN address is valid.\n"
```

Comment:
```suggestion
            "\nReturn information about the given ZEN address.\n"
```
Original wording was OK.